tasks/gitlab_helpers.py

```diff
@@ -0,0 +1,81 @@
+"""
+Helper for generating links to CI Visibility for Gitlab CI jobs
+"""
+
+import json
+import os
+from urllib.parse import quote
+
+from invoke import task
+
+CI_VISIBILITY_URL = "https://app.datadoghq.com/ci/pipeline-executions"
+
+
+@task
+def generate_ci_visibility_links(_ctx, output: str | None):
+    """
+    Generate links to CI Visibility for the current job
+    usage deva gitlab.generate-ci-visibility-links
+    Generated file
+    """
+    ci_job_id = os.environ.get("CI_JOB_ID")
+    if ci_job_id is None:
+        print("CI_JOB_ID is not set, this task can run only on Gitlab, skipping...")
+        return
+    ci_job_name = os.environ.get("CI_JOB_NAME")
+    if ci_job_name is None:
+        print("CI_JOB_NAME is not set, this task can run only on Gitlab, skipping...")
+        return
```

Comment:
```suggestion
    if ci_job_id is None:
        print(color_message("CI_JOB_ID is not set, this task can run only on Gitlab, skipping...", Color.RED))
        return
    ci_job_name = os.environ.get("CI_JOB_NAME")
    if ci_job_name is None:
        print(color_message("CI_JOB_NAME is not set, this task can run only on Gitlab, skipping...", Color.RED))
        return
```
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏